firmware/controllers/trigger/trigger_central.h

```diff
@@ -32,6 +32,8 @@ class TriggerCentral {
 	void handleShaftSignal(trigger_event_e signal DECLARE_ENGINE_PARAMETER_SUFFIX);
 	int getHwEventCounter(int index);
 	void resetCounters();
+	void resetAccumSignalData();
+	bool noiseFilter(efitick_t nowNt, trigger_event_e signal DECLARE_ENGINE_PARAMETER_SUFFIX);
```

Comment:
See impl. in https://github.com/rusefi/rusefi/pull/592/commits/d13171d5988fc832be348e84bf721507024b2d48